src/pages/Overview/OverviewPage.tsx

```diff
@@ -168,17 +164,12 @@ class OverviewPage extends React.Component<OverviewProps, State> {
             <Row style={{ marginBottom: '20px', marginTop: '20px' }}>
               {this.state.namespaces.map(ns => {
                 const nbApps = ns.appsInError.length + ns.appsInWarning.length + ns.appsInSuccess.length;
-                const encodedName = encodeURIComponent(ns.name);
+                // const encodedName = encodeURIComponent(ns.name);
```

Comment:
Probably need to uncomment that.

Reply:
IIIRC the `encodeURIComponent` is needed in case we have a namespace like "hello&world"

Reply:
Namespaces will never look like that. There are tight restrictions on the characters allowed in namespace names (and any other k8s names IIRC). Only lowercase alpha-numerics and the dash ("-") characters are allowed:

```
$ kubectl create namespace 'foo&bar'
The Namespace "foo&bar" is invalid: metadata.name: Invalid value: "foo&bar": a DNS-1123
label must consist of lower case alphanumeric characters or '-', and must start and end with
an alphanumeric character (e.g. 'my-name',  or '123-abc', regex used for validation
is '[a-z0-9]([-a-z0-9]*[a-z0-9])?')
```

Reply:
I don't know if it's needed or not but I switched back to using the encoded namespace name.